src/routes/catalog/[chatId]/+page.svelte

```diff
@@ -0,0 +1,89 @@
+<script>
+    import { writable} from 'svelte/store'
+    import { page } from '$app/stores';
+    import {listAllImages, downloadSelected } from '../../../database/catalog';
+	import { onMount } from 'svelte';
+
+let imageUrls = writable([]);
+
+$: chatId = $page.params.chatId;
+    onMount(async () => {
+    const urls = await listAllImages(chatId);
+    imageUrls.set(urls);
+});
+
+
+</script>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<div class="grid-container">
+    {#each $imageUrls as imageObj}
+      <div>
+        <figure>
+          <label class="image-wrapper">
+            <input type="checkbox" class="image-checkbox" image-url={imageObj.imageUrl}, image-title={imageObj.imageTitle}>
+            <img src={imageObj.imageUrl} alt={imageObj.imageTitle}>
+          </label>
+          <figcaption>{imageObj.imageTitle}</figcaption>
+        </figure>
+      </div>
+    {/each}
+  </div>
+<button on:click={downloadSelected} id='downloadSelected'> Download Selected</button>
```

Comment:
This button could use some styling. I would maybe center it, and also add classes "btn btn-primary" which are defined in app.css and will make this button look like all other buttons in the app